### ISSUE TITLE
Check compatibility with ilyakaznacheev/cleanenv

### ISSUE THREAD

**Issue by myasnikovdaniil** [g4s8/envdoc#28]
Hi!

Could you check the compatibility with the [github.com/ilyakaznacheev/cleanenv](https://github.com/ilyakaznacheev/cleanenv) module. It seems to me that the env syntax definition is similar to github.com/caarlos0/env.

**Comment by g4s8** [comment]
Good idea, thanks for sharing. Could you please provide any working env config type, preferably with all env related features?

**Comment by g4s8** [comment]
Good idea, thanks for sharing. Could you please provide any working env config type, preferably with all env related features?

**Comment by g4s8** [comment]
@myasnikovdaniil now you can check cleanenv support in [v1.4.0](https://github.com/g4s8/envdoc/releases/tag/v1.4.0) version
